https://issues.redhat.com/browse/ACM-29392 (#8581)
open-cluster-management/rhacm-docs: clusters/cluster_api/install_capi_managed.adoc

.Prerequisites

- You need the `MultiClusterEngine` resource either from the {acm-short} installation, or the {mce-short} standalone installation.
- The following API base domain must point to the static API VIP: `api.<cluster_name>.<base_domain>`.
- The following application base domain must point to the static IP address for Ingress VIP:

+
----
*.apps.<cluster_name>.<base_domain>
----

- You need the internal API endpoint for the cluster: `api-int.<baseDomain>`.

> - The Cluster API Provider OpenShift Assisted requires the following components:

Author: @rccrdpccl thanks for your initial review. During my peer review, I was reminded to make sure we are using the correct product names. I searched for "Cluster API Provider OpenShift Assisted" in the official product list portal and there were no results. Can you share the official product name? Is this product available yet?

Reviewer: The official name is `Cluster API Provider OpenShift Assisted`, however it's a "subproduct" (if I can call it that way) of Assisted Installer

+

> * Cluster API (CAPI): The core components provide the required custom resource definitions and controllers within the management cluster.

Reviewer 2: please remove CAPI https://cluster-api.sigs.k8s.io/

> * Cluster API (CAPI): The core components provide the required custom resource definitions and controllers within the management cluster.

Author: ```suggestion
* Cluster API: The core components provide the required custom resource definitions and controllers within the management cluster.
```

> * `assisted-service`: The {infra} in {mce} provisions this service. Ensure the service is reachable and active.

Author: ```suggestion
* Assisted service: The {infra} in {mce} provisions this service. Ensure the service is reachable and active.
```

> * `assisted-service`: The {cim} service in {mce} provisions this service. Ensure the service is reachable and active.

Author: {infra} == infrastructure operator for Red Hat OpenShift
```suggestion
* `assisted-service`: The {infra} in {mce} provisions this service. Ensure the service is reachable and active.
```

> * Assisted service: The Infrastructure Operator in {mce} provisions this service. Ensure the service is reachable and active.

Reviewer 2: Please refer to the conref for line 27

Reviewer 2: Check naming guidance, as well, same for line 28.

> * Assisted service: The Infrastructure Operator in {mce} provisions this service. Ensure the service is reachable and active.

Author: ```suggestion
* `assisted-service`: The {cim} service in {mce} provisions this service. Ensure the service is reachable and active.
```
